front/components/assistant/conversation/input_bar/toolbar/Toolbar.tsx

```diff
@@ -0,0 +1,119 @@
+import {
+  BoldIcon,
+  cn,
+  CodeBlockIcon,
+  CodeSlashIcon,
+  HeadingIcon,
+  ItalicIcon,
+  ListCheckIcon,
+  ListOrdered2Icon,
+  QuoteTextIcon,
+  Separator,
+  XMarkIcon,
+} from "@dust-tt/sparkle";
+import type { Editor } from "@tiptap/react";
+
+import { ToolbarIcon } from "@app/components/assistant/conversation/input_bar/toolbar/ToolbarIcon";
+import { ToolbarLink } from "@app/components/assistant/conversation/input_bar/toolbar/ToolbarLink";
+
+interface ToolbarProps {
+  editor: Editor | null;
+  className?: string;
+  toggleToolbar?: () => void;
+}
+
+export function Toolbar({ editor, className, toggleToolbar }: ToolbarProps) {
+  if (!editor) {
+    return null;
+  }
+
+  return (
+    <div className={cn("flex flex-col", className)}>
+      <div
+        className={cn(
+          "flex flex-row gap-2 self-end rounded-2xl bg-white px-3 py-2",
+          !toggleToolbar && "m-2"
```

Comment:
let's use `mt-2` instead, I think it looks a bit better

Reply:
ideally we want to remove pt classes too so that the input bar doesn't go too big, but if it's complex to conditionally do that let's do only mt-2 change 🙏

 https://github.com/dust-tt/dust/blob/0e85b3e50abc1eb92b3d59466483010d99a67551/front/components/assistant/conversation/input_bar/InputBarContainer.tsx#L510